src/sentry/tasks/collect_project_platforms.py

```diff
@@ -39,5 +39,5 @@ def collect_project_platforms(**kwargs):
 
     # remove (likely) unused platform associations
     ProjectPlatform.objects.filter(
-        last_seen__lte=now - timedelta(days=30),
+        last_seen__lte=now - timedelta(days=90),
```

Comment:
We don't do queries on `last_seen`. The queries are on `project_id` and `platform` but we use the `last_seen` values, so I don't think an index would help.